### PR TITLE
Rubocop

### PR DESCRIPTION
**Pull request by Middlebrooks314**
Added Rubocop into the project with the 8th light .rubocop.yml specifications, and configured the rake file to run with Travis CI.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,3 @@
-# Dont track these files on git
 
+# Dont track content of these folders
```
Comment (cmvandrevala) [inline .gitignore:1]: This file might have been changed by mistake?

File: Rakefile [hunk 2]
```diff
@@ -1,5 +1,14 @@
...
 task :test do
-  sh "bundle exec rspec"
+  sh 'bundle exec rspec'
 end
+
+task test: :rubocop
+
+# task :rubocop do
+#   sh 'rubocop'
+# end
```
Comment (cmvandrevala) [inline Rakefile]: If this rake task is not used, delete it rather than commenting it out.

**Review by cmvandrevala — CHANGES_REQUESTED** [review]
Can you make it so that rubocop runs in CI as well?

File: .gitignore (hunk 1 above)
Comment (Middlebrooks314) [reply .gitignore:1]: Yes, I checked out a new branch from master before the last PR was approved. I resolved the merge conflicts now.

File: Rakefile (hunk 2 above)
Comment (Middlebrooks314) [reply Rakefile]: I will delete it, I was confirming that the script I wrote from the rubocop docs worked before deleting the first one I found.

File: lib/game.rb [hunk 3]
```diff
@@ -1,14 +1,15 @@
+# frozen_string_literal: true
+# this is the game class
```
Comment (aaizuss) [inline lib/game.rb]: This comment is redundant - we can see that it's the game class

File: lib/game.rb [hunk 4]
```diff
@@ -1,14 +1,15 @@
+# frozen_string_literal: true
```
Comment (aaizuss) [inline lib/game.rb]: What is this for?

**Review by aaizuss — APPROVED** [review]
Looks good to me except I don't understand the `frozen_string_literal` comment on each file

**Review by cmvandrevala — APPROVED** [review]

File: lib/game.rb (hunk 3 above)
Comment (Middlebrooks314) [reply lib/game.rb]: I will delete these, they were to try and get the rubocop errors to go away.

**Pull request merged**